README.md

```diff
@@ -384,8 +384,20 @@ Requests permission to track the user. Requires an [`NSUserTrackingUsageDescript
 
 ```js
 const trackingStatus = await AdSettings.requestTrackingPermission();
-if (trackingStatus === 'authorized' || trackingStatus === 'unavailable')
+if (trackingStatus === 'authorized' || trackingStatus === 'unavailable') {
   AdSettings.setAdvertiserIDCollectionEnabled(true);
+  AdSettings.setAdvertiserTrackingEnabled(true);
+}
+```
+
+> Requires iOS 14. On Android and iOS versions below 14, this will always be no-op.
+
+### setAdvertiserTrackingEnabled
```

Comment:
Let's add a clear notice that FB won't display adds when this is not set to true